### PR TITLE
Update compress.bash

### PR DESCRIPTION
**Pull request by Qiangong2**
ColorOS

### CODE REVIEW

File: compress.bash [hunk 1]
```diff
@@ -11,19 +11,19 @@
...
 
-name=AOSP
+name=ColorOS
 
 
 # Manifest link. https:// is mandatory.
 # Example: https://github.com/cyanogenmod/android
 
-manifest=https://android.googlesource.com/platform/manifest
+manifest=https://github.com/Qiangong2/patchrom/default.xml
```
Comment (msfjarvis) [inline compress.bash:20]: Don't make me call you a moron please. Just don't

File: compress.bash [hunk 2]
```diff
@@ -34,12 +34,12 @@ branch=7.0.0_r6
 # compressrepo
 # If true, will compress the .repo folder.
 
-export compressrepo=true
+export compressrepo=false
```
Comment (msfjarvis) [inline compress.bash:37]: Leave it at `true`

**Review by msfjarvis — CHANGES_REQUESTED** [review]
Needs to address the issues I have raised before it will be merged

**Comment by regalstreak** [comment]
Server is down and im too lazy to fix it. @MSF-Jarvis if youve got yours running do this maybe

**Comment by msfjarvis** [comment]
@regalstreak We are not responsible for fixing shit PRs

**Comment by regalstreak** [comment]
I mean, if the pr is merged, take over for now.

**Comment by msfjarvis** [comment]
@regalstreak Not merged and won't be

**Pull request closed without merging**